src/org/omegat/Bundle.properties

```diff
@@ -698,9 +698,9 @@ CT_STATS_Segments=Segments
 
 CT_STATS_Words=Words
 
-CT_STATS_Characters_NOSP=Characters (without spaces)
+CT_STATS_Characters_NOSP=Characters (w/o spaces)
 
-CT_STATS_Characters=Characters (including spaces)
+CT_STATS_Characters=Characters (+ spaces)
```

Comment:
I feel like "+" is confusing in contrast to "w/o". It makes me think that the number is not simply characters *including* spaces, but that we are "adding" spaces, or something (even if that logically doesn't make sense, it creates doubt in my mind.)

I would prefer "w/" as an abbreviation for "with".

Reply:
You're right. Thank you.